src/main/scala/com/wavesplatform/Coordinator.scala

```diff
@@ -199,6 +199,10 @@ object Coordinator extends ScorexLogging with Instrumented {
       blockGs = blockData.generationSignature.arr
       _ <- Either.cond(calcGs.sameElements(blockGs), (), s"declared generation signature ${blockGs.mkString} does not match calculated generation signature ${calcGs.mkString}")
       effectiveBalance <- genBalance(height)
+      _ <- Either.cond(blockTime < fs.minimalGeneratingBalanceAfter ||
```

Comment:
this has been moved to `validateEffectiveBalance`, see `genBalance` in previous line